From c091abfca6c51079478431539b56df0e0d0346b1 Mon Sep 17 00:00:00 2001
From: zhangguangzhi <zhangguangzhi3@huawei.com>
Date: Wed, 18 Sep 2019 03:27:22 -0400
Subject: [PATCH] fix test error g2-1.15


diff --git a/test/results/add/g2-1.15 b/test/results/add/g2-1.15
index b67c373..ee5f868 100644
--- a/test/results/add/g2-1.15
+++ b/test/results/add/g2-1.15
@@ -82,7 +82,7 @@ menuentry 'Fedora 21 Rescue' --class fedora --class gnu-linux --class gnu --clas
 	  search --no-floppy --fs-uuid --set=root 6169b46f-0257-4319-b2e4-caaed2a8e06b
 	fi
 	linuxefi /vmlinuz-0-rescue-5a94251776a14678911d4ae0949500f5 root=/fooooo ro  rd.lvm.lv=fedora_uefi/root rd.lvm.lv=fedora_uefi/swap rhgb quiet
-	initrdefi /initramfs-0-rescue-5a94251776a14678911d4ae0949500f5.img
+	initrd /initramfs-0-rescue-5a94251776a14678911d4ae0949500f5.img
 }
 menuentry 'Fedora, with Linux 3.15.0-0.rc5.git2.10.fc21.x86_64' --class fedora --class gnu-linux --class gnu --class os --unrestricted $menuentry_id_option 'gnulinux-3.15.0-0.rc5.git2.10.fc21.x86_64-advanced-a14e3dcb-ade3-42f7-832f-d9f66b5ae6a3' {
 	load_video
-- 
2.19.1

